plenum/persistence/req_id_to_txn.py

```diff
@@ -65,3 +66,7 @@ def size(self):
 
     def close(self):
         self._keyValueStorage.close()
+
+    @staticmethod
+    def _full_digest_key(digest: str):
```

Comment:
I would add a unit test to write same `payload_digest` and `full_digest` and making sure that they both are available and not override.